databricks/koalas/series.py

```diff
@@ -4873,10 +4873,11 @@ def item(self):
         >>> kser.item()
         10
         """
-        item_top_two = self[:2]
```

Comment:
I think you can just simply fix this line to `self[:2].to_pandas()`

Reply:
Actually, why don't you just remove all here and call `self[:2].to_pandas().item()`?

Reply:
Also maybe we should use head(2) instead of [:2]

Reply:
Thank you guys!